app/src/main/java/com/kickstarter/ui/viewholders/CommentCardViewHolder.kt

```diff
@@ -7,6 +7,7 @@ import com.kickstarter.libs.utils.DateTimeUtils
 import com.kickstarter.models.Comment
 import com.kickstarter.ui.views.OnCommentCardClickedListener
 import com.kickstarter.viewmodels.CommentsViewHolderViewModel
+import kotlinx.android.synthetic.main.comment_card.view.*
```

Comment:
Wildcard imports are forbidden by the ktlinter, I'll check why the linter has not marked this as an error, but please change the wildcard import